pkg/apis/core/validation/validation.go

```diff
@@ -4555,6 +4555,7 @@ func ValidateResourceRequirements(requirements *core.ResourceRequirements, fldPa
 // validateResourceQuotaScopes ensures that each enumerated hard resource constraint is valid for set of scopes
 func validateResourceQuotaScopes(resourceQuotaSpec *core.ResourceQuotaSpec, fld *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
+	glog.V(4).Infof("scopes %v", resourceQuotaSpec.Scopes)
```

Comment:
nit: do you really want to leave this glog msg?

Reply:
I was thinking that this log message will not repeat again and again and will not spam the log file. OTOH, it might be useful in some misconfiguration cases.

Reply:
I am fine if you want to keep it, but please raise the verbosity levels to at least 6 and give it more descriptive text.